cli/script/command-executor.ts

```diff
@@ -1289,5 +1296,16 @@ function whoami(command: cli.ICommand): Promise<void> {
     return sdk.getAccountInfo()
         .then((account): void => {
             log(`${account.email} (${account.linkedProviders.join(", ")})`);
```

Comment:
What do you think about prefixing this line with "Account: " if there is a proxy? That way we display two sets of key-value pairs that are clearly distinguished.